src/components/utils/src/logger_qt.cc

```diff
@@ -133,8 +134,8 @@ bool logger::Logger::Impl::InitLogger(const bool logs_enabled,
 }
 
 void logger::Logger::Impl::DeinitLogger() {
-  CREATE_LOGGERPTR_LOCAL(logger_, "Logger");
-  LOGGER_DEBUG(logger_, "Logger deinitialization");
+  CREATE_LOGGERPTR_LOCAL("Logger");
```

Comment:
@OHerasym I think `CREATE_LOGGERPTR_LOCAL` macro could be renamed somehow to follow common log template (SDL word should be present)